server/routes/items.js

```diff
@@ -5,8 +5,15 @@ const router = require("express").Router();
 const items = require("../data/items.json");
 
 router.get("/api/items/:category", (req, res) => {
+  // Normally stored on a db
+  const validCategories = ["lifestyle", "fitness", "medical", "entertainment"];
```

Comment:
Missing the "industrial" and "pets and animals" categories. can you add, please?

Reply:
Thanks! I will add it